tests/UserLogin.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Tests;
+
+use App\Models\User;
+
+trait UserLogin{
+
+    public $user;
+
+    public function setUpUser(){
+
+        $this->user = User::factory()->create();
+
+        $response = $this->post('/login', [
+            'email' => $this->user->email,
+            'password' => 'password',
+        ]);
+        $this->assertAuthenticated();
+    }
```

Comment:
each time the login route call is not required. you can put common method on pest.php